modules/services/eks/main.tf

```diff
@@ -0,0 +1,8 @@
+// TODO: Check if it already exists (in case if the customer runs this script twice)
```

Comment:
terraform should take care of this, shouldn't it? (I mean running the script twice)

Reply:
I tested that locally and TF will return an error saying that access entry already exists. That error didn't look pretty, so I thought maybe adding some logic to do the check would be a better customer experience. However, I don't see this as a critical improvement.

Reply:
Anyway, I have removed the comment.